synapse/lib/node.py

```diff
@@ -511,3 +511,113 @@ def fork(self, node):
         nodes.append(node)
 
         return Path(self.vars, nodes)
+
+def props(node):
```

Comment:
i've been using "pode" for a packed node, might go well here to avoid confusion over node being a Node()?

Reply:
sure - quick change